pyproject.toml

```diff
@@ -1,9 +1,20 @@
 [tool.poetry]
 name = "temporalio"
-version = "0.1.0"
+version = "0.1a1.dev1"
 description = "Temporal.io Python SDK"
 authors = ["Temporal Technologies Inc <sdk@temporal.io>"]
 license = "MIT"
+# We need to include proto source that is otherwise excluded via .gitignore.
```

Comment:
I still wish we didn't exclude proto files from being in-repo. It it so annoying for a user, if they want to see the proto types/files, to have to download a prebuilt artifact and extract it because we won't commit the source.

Reply:
They're in-repo via the core submodule though, no?